manim/mobject/three_d/three_dimensions.py

```diff
@@ -1183,8 +1187,8 @@ def __init__(
             height=height,
             **kwargs,
         )
-        self.cone.shift(end)
-        self.end_point = VectorizedPoint(end)
+        self.cone.shift(np.array(end, np.float64))
+        self.end_point = VectorizedPoint(np.array(end, np.float64))
```

Comment:
Maybe the following would be better to reduce copying
```suggestion
        np_end = np.asarray(end, dtype=np.float64)
        self.cone.shift(np_end)
        self.end_point = VectorizedPoint(np_end)
```

Reply:
I will add proposed changes today. Thanks for fast review.